src/Handler.php

```diff
@@ -12,24 +12,85 @@ class Handler
     protected $client;
 
     /**
+     * The previously registered error handler as returned by the interpreter.
+     *
+     * @var callable|null
+     */
+    protected $previousErrorHandler;
+
+    /**
+     * The previously registered exception handler as returned by the interpreter.
+     *
+     * @var callable|null
+     */
+    protected $previousExceptionHandler;
+
+    /**
      * Register our exception handler.
      *
      * @param \Bugsnag\Client|string|null $client client instance or api key
+     * @param bool $callPrevious whether or not to call the previous handlers
      *
      * @return static
      */
-    public static function register($client = null)
+    public static function register($client = null, $callPrevious = true)
```

Comment:
> Why would optional parameters that preserve the existing behavior break backwards compatibility? [obviously this does not preserve existing behavior, but we can continue that discussion above]

Because PHP allows you to call methods with too many parameters without errors, and this class is non-final so could have been extended with a different signature for the 2nd parameter. Given the current handler didn't support preserving the previous handler, I'm sure other people will have extended it in their projects to add this.